data/sites.json

```diff
@@ -17,7 +17,7 @@
     "tagline": "The easiest way to get paid in Bitcoin",
     "description": "",
     "website": "https://getpaidinbitcoin.com.au",
-    "logo": "https://images.squarespace-cdn.com/content/v1/5f5990cc9e792617828214ae/1600921877148-3ISFDDENM1N89GLGAIKI/GBIB+Logo_5%402x.png?format=1500w",
+    "logo": "https://images.squarespace-cdn.com/content/v1/5f5990cc9e792617828214ae/1600921877148-3ISFDDENM1N89GLGAIKI/GBIB+Logo_5%402x.png",
```

Comment:
The image needs to be in our repo

Reply:
This feedback does not relate to the ticket. Happy to do this but I think it should be scoped out in another ticket. :) @bitcoinbrisbane

Reply:
Ok, other vendor